ducktape/tests/test.py

```diff
@@ -137,3 +129,168 @@ def is_test_class(self, obj):
         return inspect.isclass(obj) and issubclass(obj, Test) and len(obj.__subclasses__()) == 0
 
 
+class TestReporter(object):
```

Comment:
You'll probably just want to move these classes into their own files, at least by type (Reporters, Loaders, etc).